runtime/plugin/man.vim

```diff
@@ -1,6 +1,21 @@
-if get(g:, 'loaded_man', 0)
+if exists('g:loaded_man')
   finish
 endif
 let g:loaded_man = 1
 
-command! -count=0 -nargs=+ Man call man#get_page(<count>, <f-args>)
+if !exists("g:find_man_window")
+  let g:find_man_window = 1
+endif
+
+if !exists('g:man_synopsis')
+  let g:man_synopsis = 'SYNOPSIS'
+endif
+
+command! -bang -complete=customlist,man#Complete -nargs=* Man call
+      \ man#get_page(<bang>0, 'edit', <f-args>)
+command! -bang -complete=customlist,man#Complete -nargs=* Sman call
+      \ man#get_page(<bang>0, 'split', <f-args>)
+command! -bang -complete=customlist,man#Complete -nargs=* Vman call
+      \ man#get_page(<bang>0, 'vsplit', <f-args>)
+command! -bang -complete=customlist,man#Complete -nargs=* Tman call
+      \ man#get_page(<bang>0, 'tabe', <f-args>)
```

Comment:
Of all these commands only `:Man` is definitely needed. `:Sman` may be needed to avoid side-effects of running `:split` before `:Man` (AFAIK you will at least have a few \*Win\* autocommands with unexpected patterns), but others should be possible to emulate using modifiers like `:vertical` or `:tab`.